storage.tf

```diff
@@ -1,31 +1,32 @@
-  engine_version               = var.postgresql_version
-  instance_class               = var.postgresql_instance_class
-  name                         = var.postgresql_db_name
-  username                     = var.postgresql_username
-  password                     = var.postgresql_password
-  parameter_group_name         = aws_db_parameter_group.main.name
-  db_subnet_group_name         = aws_db_subnet_group.main.name
-  deletion_protection          = var.postgresql_deletion_protection
-  multi_az                     = var.postgresql_multi_az
-  port                         = var.postgresql_port
-  copy_tags_to_snapshot        = var.postgresql_copy_tags_to_snapshot
-  storage_encrypted            = var.postgresql_storage_encrypted
-  kms_key_id                   = aws_kms_key.main.arn
-  vpc_security_group_ids       = [aws_security_group.firewall_rule.id]
-  final_snapshot_identifier    = var.postgresql_name
-  backup_retention_period      = var.postgresql_backup_retention_period
-  backup_window                = var.postgresql_backup_window
-  replicate_source_db          = var.postgresql_replicate_source_db
-  publicly_accessible          = var.postgresql_publicly_accessible
-  performance_insights_enabled = var.postgresql_performance_insights_enabled
+  apply_immediately               = var.postgresql_apply_immediately
+  identifier                      = var.postgresql_name
+  allocated_storage               = var.postgresql_allocated_storage
+  storage_type                    = var.postgresql_storage_type
+  engine                          = "postgres"
+  engine_version                  = var.postgresql_version
+  instance_class                  = var.postgresql_instance_class
+  name                            = var.postgresql_db_name
+  username                        = var.postgresql_username
+  password                        = var.postgresql_password
+  parameter_group_name            = aws_db_parameter_group.main.name
+  db_subnet_group_name            = aws_db_subnet_group.main.name
+  deletion_protection             = var.postgresql_deletion_protection
+  multi_az                        = var.postgresql_multi_az
+  port                            = var.postgresql_port
+  copy_tags_to_snapshot           = var.postgresql_copy_tags_to_snapshot
+  storage_encrypted               = var.postgresql_storage_encrypted
+  kms_key_id                      = aws_kms_key.main.arn
+  vpc_security_group_ids          = [aws_security_group.firewall_rule.id]
+  final_snapshot_identifier       = var.postgresql_name
+  backup_retention_period         = var.postgresql_backup_retention_period
+  backup_window                   = var.postgresql_backup_window
+  replicate_source_db             = var.postgresql_replicate_source_db
+  publicly_accessible             = var.postgresql_publicly_accessible
+  performance_insights_enabled    = var.postgresql_performance_insights_enabled
+  enabled_cloudwatch_logs_exports = ["postgresql"]
```

Comment:
@manutarus is this parameter something we also want to enable for the "from-snapshot" RDS instances?

Reply:
@jasonrogena I have pushed a commit that add the parameter for the "from-snapshot"  instances as well